Update tutorial-custom-docker-image.md

petrepopescu21 opened the pull request:
Corrected webssh step: The "service ssh start" command needs to be run from a shell script and not a Dockerfile RUN command.

azurecla commented:
Hi __@fmigacz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@fmigacz Thank you for the contribution!  I've sent the change to the author for review.

cherylmc commented:
Thanks for contribution! Change approved.

petrepopescu21 committed:
Update tutorial-custom-docker-image.md

Corrected webssh step: The "service ssh start" command needs to be run from a shell script and not a Dockerfile RUN command.

petrepopescu21 committed:
Updated reqs for starting the webssh service

Fixed formatting

GitHubber17 committed:
Update tutorial-custom-docker-image.md
